### ISSUE TITLE
lua pipe child close() socket failed (9: Bad file descriptor)

### ISSUE THREAD

**Issue by theMiddleBlue** [Kong/kong-python-pdk#1]
Hi,

I'm testing the py-hello plugin on my kong and it works correctly! The only problem is that I see a ton of errors on the nginx error log like:

```
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8000 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8000 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8000 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8000 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8000 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8443 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8443 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8443 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8443 failed (9: Bad file descriptor)
2021/03/24 17:56:45 [emerg] 3017#0: lua pipe child close() socket 0.0.0.0:8443 failed (9: Bad file descriptor)
```

have you got any idea of why?

Thanks!

**Comment by fffonion** [comment]
Probably related to https://github.com/Kong/kong/issues/6948

**Comment by fffonion** [comment]
@theMiddleBlue please see my comment in the above linked issue in Kong, it will be fixed in kong 2.4 and currently has no side effect to Kong expect from noisy logs.

**Comment by fffonion** [comment]
Closing this as it's fixed in 2.4, please reopen or create new issue if you are still facing the issue.
